src/Resources/src/api/getParticipationsByDay.ts

```diff
@@ -0,0 +1,45 @@
+import useApi from '@/api/api';
+import { onMounted, onUnmounted, readonly, ref } from 'vue';
+
+export type ListData = [];
+
+const listDataState = ref([]);
+
+/**
+ * if date is passed participants list is specific to that date, if not it returns the list of today's participants
+ * @param date
+ * @returns list of participants
+ */
+export function useParticipationsListData(date: string) {
+    const loaded = ref(false);
+    let useParticipationsError = false;
```

Comment:
wenn in einer Vue Komponente auf den Wert einer Variable reagiert werden soll, muss diese als reaktive Vue Variable definiert werden (ref(), reactive(), computed(),...) sonst kann es zu unerwartetem Verhalten kommen.